### PR TITLE
added a password_manager app. in python section.

### LINKED ISSUE #183: password management app
i wanted to add a python project 
a password management app.
i hoping if could add it here.

**Languages**

-   [x] Python

### PR DESCRIPTION
**Pull request by shaalwan**
# Pasword-Manager

added a password management app in python. python needs to be installed in you system to run this application. 
pyperclip module is required 
To install this module type command in terminal/command prompt 
                                    
                                           pip install pyperclip

## Type of change

-   [x] New feature (non-breaking change which adds functionality)

# Checklist:

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by kaustubhgupta** [comment]
@shaalwan kindly fill the PR template properly with all entries

File: Python/password-manager.pyw [hunk 1]
```diff
@@ -0,0 +1,151 @@
+# imports
```
Comment (kaustubhgupta) [inline Python/password-manager.pyw]: Remove extra spaces from the program

File: Python/password-manager.pyw [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+Label(root, text="      Enter length of Password : ",
+      font=("Aerial Bold", 14)).grid(row=2, column=0)
+
+Label(root, text="(Password must be of atleast 8 characters)",
+      font=("Sans Serif", 10)).grid(row=3, column=1, columnspan=2)
+
+
+# takes value from user
+e = IntVar()
```
Comment (kaustubhgupta) [inline Python/password-manager.pyw]: User better variable name here

File: Python/password-manager.pyw [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+    Button(root, text="Generate", height=3, width=18, bd=8, activebackground="blue", command=res,
+           relief="raised", bg="white", state="disabled", fg="blue").grid(row=30, column=60,)
+
+
+button = Button(root, text="Generate", height=3, width=18, bd=8, activebackground="blue",
+                command=res, relief="raised", bg="white", fg="blue").grid(row=30, column=60,)
+
+
+root.mainloop()  # loop that allows GUI to run
```
Comment (kaustubhgupta) [inline Python/password-manager.pyw]: - The code is fine, do the changes suggested in the beginning. Also, attach a GIF showing the working of the program, its functions as right now it's difficult to judge whether it is working or not. Put both the files (GIF and code file in a folder and put that folder under the Python directory. 
- Update the language folder readme too :-)

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by shaalwan** [comment]
i did changes and pushed them to shaalwan branch. please look into it. thank you!

**Review by kaustubhgupta — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
@shaalwan Thanks for contribution!

**Pull request merged**